### PR TITLE
[zh-cn]: create the translation of Navigator `getVRDisplays()` method

### PR DESCRIPTION
**Pull request by T34-active**
### Description
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getVRDisplays

### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/navigator/getvrdisplays/index.md
* Last commit: https://github.com/mdn/content/commit/9a9dcd5848aaf1c4226f746539e6218c48647249

### CODE REVIEW

File: files/zh-cn/web/api/navigator/getvrdisplays/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Navigator：getVRDisplays() 方法
+slug: Web/API/Navigator/getVRDisplays
+l10n:
+  sourceCommit: null
```
Comment (T34-active) [inline files/zh-cn/web/api/navigator/getvrdisplays/index.md:5]:
```suggestion
  sourceCommit: 9a9dcd5848aaf1c4226f746539e6218c48647249
```

File: files/zh-cn/web/api/navigator/getvrdisplays/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: Navigator：getVRDisplays() 方法
+slug: Web/API/Navigator/getVRDisplays
+l10n:
+  sourceCommit: 9a9dcd5848aaf1c4226f746539e6218c48647249
+---
+
+{{APIRef("WebVR API")}}{{Deprecated_Header}}{{Non-standard_Header}}
+
+{{domxref("Navigator")}} 接口的 **`getVRDisplays()`** 方法会返回一个兑现为 {{domxref("VRDisplay")}} 对象数组的 promise，该数组代表连接到计算机的任何可用 VR 显示器。
```
Comment (T34-active) [inline files/zh-cn/web/api/navigator/getvrdisplays/index.md:10]:
```suggestion
{{domxref("Navigator")}} 接口的 **`getVRDisplays()`** 方法会返回兑现为 {{domxref("VRDisplay")}} 对象数组的 promise，该数组代表连接到计算机的任何可用 VR 显示器。
```

File: files/zh-cn/web/api/navigator/getvrdisplays/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: Navigator：getVRDisplays() 方法
+slug: Web/API/Navigator/getVRDisplays
+l10n:
+  sourceCommit: 9a9dcd5848aaf1c4226f746539e6218c48647249
+---
+
+{{APIRef("WebVR API")}}{{Deprecated_Header}}{{Non-standard_Header}}
+
+{{domxref("Navigator")}} 接口的 **`getVRDisplays()`** 方法会返回兑现为 {{domxref("VRDisplay")}} 对象数组的 promise，该数组代表连接到计算机的任何可用 VR 显示器。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/getvrdisplays/index.md:10]:
```suggestion
{{domxref("Navigator")}} 接口的 **`getVRDisplays()`** 方法会返回兑现为 {{domxref("VRDisplay")}} 对象数组的 promise，该数组代表连接到计算机的任何可用的 VR 显示器。
```

File: files/zh-cn/web/api/navigator/getvrdisplays/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+兑现为 {{domxref("VRDisplay")}} 对象数组的 promise。
+
+## 示例
+
+有关示例代码，请参见 [`VRDisplay`](/zh-CN/docs/Web/API/VRDisplay#examples)。
+
+## 规范
+
+此方法是旧版 [WebVR API](https://immersive-web.github.io/webvr/spec/1.1/) 的一部分，已被 [WebXR 设备 API](https://immersive-web.github.io/webxr/) 取代。其不再处于标准化轨道。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/getvrdisplays/index.md:32]:
```suggestion
此方法属于旧版的 [WebVR API](https://immersive-web.github.io/webvr/spec/1.1/)（已被 [WebXR 设备 API](https://immersive-web.github.io/webxr/) 取代）。其不再处于标准化轨道。
```

**Review by yin1999 — APPROVED** [review]
Thank you @T34-active

**Pull request merged**